jupyter-notebooks/Supplemental_PAO1_FastICA_example.ipynb

```diff
@@ -0,0 +1,1467 @@
+      "132 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "117 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "105 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "106 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "128 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "81 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "131 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "107 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "107 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "130 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "119 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "130 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "130 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "83 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "131 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "132 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "118 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "121 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "83 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "105 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "145 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "128 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probabilities'\n",
+      "82 adjustments made to a vector of length 186 containing the raw weight values in a call to 'update_probab
```

Comment:
What is the output? Unclear to me what this is doing - perhaps renaming the function and output variable will help with understanding